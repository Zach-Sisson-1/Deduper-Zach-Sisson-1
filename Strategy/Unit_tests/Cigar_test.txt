354M218N187M
1S354M218N187M1S
11S354M218N187M
11S354M218N187M
72M
1S71M
71M1S
70M2S
41M53N31M
6S66M
71M1S
2S70M
68M4S
3S69M
1S49M1I21M