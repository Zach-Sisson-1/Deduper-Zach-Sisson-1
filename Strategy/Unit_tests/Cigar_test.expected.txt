759
760
759
759
72
71
72
72
125
66
72
70
72
69
70